### ISSUE TITLE
not getting custom/standard notifications

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#931]
On Thursday 3/12 I set up standard and custom notifications for new event in a particular state, updated event in a particular state, and an event I was a collaborator on and I didn't get any in-app notifications nor any emails (and I had the email option clicked on).
![image](https://user-images.githubusercontent.com/49287076/76795388-0a5fe200-6797-11ea-9d1c-3d26578ce302.png)

I have step-by-step details of what I did if you want it but it seems to be blanket problem (with Celery Beat?) of notifications not going through so I'll spare you the details unless you want them.

**Comment by JChipault** [comment]
@nbaertlein  per your recommendation, I checked the notifications in the test API and I did not see them listed there either.

**Comment by BlakeDraper** [comment]
@JChipault You are correct that this is nearly certainly a server configuration issue for the non-real-time notifications (Celery Beat) - so while them not currently working is bad news, the better news is that they should all work at once, once that server issue is addressed.
